src/file/file.controller.ts

```diff
@@ -37,6 +39,26 @@ export class FileController {
 
   constructor(private fileService: FileService) {}
 
+  private getContentType = (contentType: string, acceptHeader: string) => {
```

Comment:
what's the purpose of arrow function here? shouldn't it be just private method for the sake of consistency?

Reply:
You're right, updating :)